packages/expo-modules-core/ios/Fabric/ExpoFabricView.swift

```diff
@@ -53,18 +67,18 @@ public class ExpoFabricView: ExpoFabricViewObjC {
       // TODO: @tsapeta: Figure out better way to rethrow errors from here.
       // Adding `throws` keyword to the function results in different
       // method signature in Objective-C. Maybe just call `RCTLogError`?
-      try? prop.set(value: Conversions.fromNSObject(newValue), onView: view, appContext: context)
+      try? prop.set(value: Conversions.fromNSObject(newValue), onView: self, appContext: context)
     }
   }
 
   /**
    Calls lifecycle methods registered by `OnViewDidUpdateProps` definition component.
    */
   public override func viewDidUpdateProps() {
-    guard let view = contentView, let viewDefinition = moduleHolder?.definition.view else {
+    guard let viewDefinition = viewDefinition else {
```

Comment:
You can skip it if the name is the same.
```suggestion
    guard let viewDefinition else {
```